.rubocop_todo.yml

```diff
@@ -796,14 +796,15 @@ Rails/FilePath:
     - 'spec/models/content_configuration_spec.rb'
     - 'spec/support/downloads_helper.rb'
 
-# Offense count: 11
+# Offense count: 12
 # Configuration parameters: Include.
 # Include: app/models/**/*.rb
 Rails/HasAndBelongsToMany:
   Exclude:
     - 'app/models/concerns/payment_method_distributors.rb'
     - 'app/models/enterprise.rb'
     - 'app/models/enterprise_group.rb'
+    - 'app/models/order_cycle.rb'
```

Comment:
:thinking: I didn't know that this was considered bad style. Maybe I wouldn't have suggested it. :wink: 

The documentation of `has_many through` actually says:

> Then new join models are automatically created for the newly associated objects. If some that existed previously are now missing, then their join rows are automatically deleted.
> 
> Automatic deletion of join models is direct, no destroy callbacks are triggered.

So that should have worked without your previous cascade declaration. Maybe there's a better way but I'm happy with the current solution because it's less code.

Reply:
:+1: 

> So that should have worked without your previous cascade declaration.

Ah good to know